src/policies/enacted_policies.py

```diff
@@ -0,0 +1,495 @@
+        Starting April 26th the Bundesnotbremse is in place. That means when counties
+        have a >165 incidence schools only offer classes to graduating classes and
+        emergency care. (https://bit.ly/3aUd2KC)
+
+        - BW: federal guidelines apply (https://bit.ly/3t7AIBJ,
+          https://bit.ly/3aR4yUM)
+        - BY:
+            - source: https://bit.ly/2RgmsJm (accessed 2021-04-30)
+            - incidence <100: A/B for everyone
+            - incidence >100: 4th grade and graduating classes in A/B schooling.
+              emergency care for rest.
+            - incidences >100 in most counties!
+        - NRW:
+            - sources: https://bit.ly/2QHWChG, https://bit.ly/3gPraZu,
+              https://bit.ly/32Zq1q8, https://bit.ly/3nzNhEx
+            - A/B schooling when incidences <165 b/c of Bundesnotbremse
+
+        As cases are falling in that time frame (on May 6 ~75% of counties
+        were below the 165 threshold, we take the more optimistic scenario that
+        schools have A/B schooling for everyone plus emergency care
+        and graduating classes.
+
+    """
+    strict_emergency_care = _get_school_strict_emergency_care()
+    generous_emergency_care = (
+        _get_school_generous_emergency_care_with_a_b_for_graduating_classes()
+    )
+    primary_and_graduating_in_ab = (
+        _get_generous_emergency_care_with_a_b_for_primary_and_graduation_classes()
+    )
+    # A/B for everyone, graduating classes attend in full + emergency care.
+    mid_march_to_easter_policy = _get_policy_mid_march_to_easter()
+
+    # combine specs
+    specs = [
+        ("before_november_2020", "2020-11-01", reduce_educ_models, 1.0),
+        ("until_christmas_2020", "2020-12-15", reduce_educ_models, HYGIENE_MULTIPLIER),
+        ("christmas-lockdown", "2021-01-10", *strict_emergency_care),
+        ("jan_and_feb_2021", "2021-02-21", *generous_emergency_care),
+        ("feb_22_to_mid_march", "2021-03-14", *primary_and_graduating_in_ab),
+        ("mid_march_to_easter", "2021-04-05", *mid_march_to_easter_policy),
+        ("easter_until_may", "2021-04-30", *generous_emergency_care),
+        ("summer_educ_policies_2021", VERY_LATE, *mid_march_to_easter_policy),
+    ]
+    start_date = VERY_EARLY
+    to_combine = []
+    for prefix, end_date, func, kwargs in specs:
+        block_info = {
+            "start_date": start_date,
+            "end_date": pd.Timestamp(end_date),
+            "prefix": prefix,
+        }
+        if isinstance(kwargs, float):
+            kwargs = {"multiplier": kwargs}
+
+        to_combine.append(
+            func(
+                contact_models=contact_models,
+                block_info=block_info,
+                educ_type="school",
+                **kwargs,
+            )
+        )
+        start_date = pd.Timestamp(end_date) + pd.Timedelta(days=1)
+    school_policies = combine_dictionaries(to_combine)
+    return school_policies
+
+
+def _get_policy_mid_march_to_easter():
```

Comment:
This cannot be called get_...._policies. Policies in sid are dictionaries that fulfill a few conditions. Any get_..._policies function must return such dictionaries.